<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Плавающий горизонт</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="funcs">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>260</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>cos(x)*cos(z)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string> x*z / (sqrt(x*x + z*z))</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>cos(x+z)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>cos(x) + cos(z)</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit_y">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>340</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>170</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>angle_z</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>100</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>angle_y</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_x_r">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>50</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_z_2">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>430</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0.2</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_x">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>340</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>angle_x</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_z">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>340</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0.2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_z_r">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>190</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>320</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>max_y</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_x_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>430</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rot_3">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>120</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Повернуть </string>
    </property>
   </widget>
   <widget class="QPushButton" name="draw">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>510</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Построить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>320</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>min_x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>410</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>d_z</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>320</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>d_x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>410</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>min_z</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_y_r">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>120</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rot">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>50</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Повернуть </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rot_2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>190</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Повернуть </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_y_2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>430</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>410</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>max_z</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>976</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
